src/collections/_documentation/platforms/react-native/touchevents.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Tracking Touch Events'
+---
+
+{% version_added 1.5.0 %}
+
+To track touch events with the React Native SDK, you will need at least version `1.5.0` and will have to wrap your app with a `TouchEventBoundary`. You can track touch events in two ways:
+
+## Wrapping with `TouchEventBoundary`
+At the root of your app, usually `App.js`, wrap the app component with `Sentry.TouchEventBoundary`:
+
+```jsx
+import * as Sentry from "@sentry/react-native";
+
+const App = () => {
+    return (
+        <Sentry.TouchEventBoundary>
+            <RestOfTheApp />
+        </Sentry.TouchEventBoundary>
+    );
+}
+
+export default AppRegistry.registerComponent("Your Amazing App", () => App);;
+```
+
+## Using the `withTouchEventBoundary` Higher-Order Component
+
+At the root of your app, usually `App.js`, wrap the app component with `Sentry.withTouchEventBoundary`:
+
+```jsx
+import * as Sentry from "@sentry/react-native";
+
+const App = () => {
+    return (
+        <RestOfTheApp />
+    );
+}
+
+export default AppRegistry.registerComponent("Your Amazing App", () => Sentry.withTouchEventBoundary(App));;
+```
+
+{% capture __alert_content -%}
+While you can track specific sections of your app by wrapping each section with the boundary, do not nest them or the touches could be tracked multiple times along with possible undefined behavior.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Multiple Boundaries"
+  content=__alert_content
+  level="warning"
+  deep_link="multiple-boundaries"
+%}
+
+## How Touches are Tracked Automatically
+Each touch event that occurs will be automatically logged as a breadcrumb, and will be displayed on the dashboard when an event occurs along with the component tree that the touch event occurred in. This component tree is logged using the `name` property of each component, which by default is the name given to it when it was defined or the name given to it by React Native such as "View" or "ScrollView".
```

Comment:
```suggestion
Each touch event that occurs will be automatically logged as a breadcrumb, and will be displayed on the dashboard when an event occurs along with the component tree that the touch event occurred in. This component tree is logged using the `name` property of a component. By default, React Native will set this property automatically on components.
```

> This component tree is logged using the `name` property of each component, which by default is the name given to it when it was defined or the name given to it by React Native such as "View" or "ScrollView".

Run on sentence, and we probably can just avoid going super detailed

As an aside did you use https://github.com/facebook/react-native/blob/3276563806966c16256181d9e71099f63f26e67f/Libraries/Renderer/implementations/ReactFabric-prod.js#L1020-L1060 when working on this?

Reply:
Ah no I didn't, I'll see if we can get access to that method. Thanks!